bindings/node.js/test.ts

```diff
@@ -0,0 +1,45 @@
+import { randomBytes } from "crypto";
+import {
+  loadTrustedSetup,
+  freeTrustedSetup,
+  blobToKzgCommitment,
+  computeAggregateKzgProof,
+  verifyAggregateKzgProof,
+  BYTES_PER_FIELD,
+  FIELD_ELEMENTS_PER_BLOB,
+  verifyKzgProof,
+} from "./kzg";
+
+const SETUP_FILE_PATH = "../../src/trusted_setup.txt";
+const BLOB_BYTE_COUNT = FIELD_ELEMENTS_PER_BLOB * BYTES_PER_FIELD;
+
+const generateRandomBlob = () => new Uint8Array(randomBytes(BLOB_BYTE_COUNT));
+
+describe("C-KZG", () => {
+  beforeAll(() => {
+    loadTrustedSetup(SETUP_FILE_PATH);
+  });
+
+  afterAll(() => {
+    freeTrustedSetup();
+  });
+
+  it("verifies a proof at a given commitment point", () => {
+    const blob = generateRandomBlob();
+    const commitment = blobToKzgCommitment(blob);
+    const proof = computeAggregateKzgProof([blob]);
+
+    // It doesn't seem to matter what is passed here...
+    const z = Uint8Array.from(new Array(32).fill(0));
+    const y = Uint8Array.from(new Array(32).fill(0));
+
+    expect(verifyKzgProof(commitment, z, y, proof)).toBe(true);
```

Comment:
`verifyKzgProof` seems to return true no matter what `z` and `y` values it is given...

Need to debug the VerifyKzgProof C++ function to figure out what's going on.